asn1ate/sema.py

```diff
@@ -25,7 +25,6 @@
 
 from asn1ate import parser
 
-
```

Comment:
pylint wants two blank lines between imports and first code entity, can you please add that back?